spp_api/models/spp_api_path.py

```diff
@@ -552,6 +555,34 @@ def _get_parameters_one_element(self):
             self._context_parameter(),
         ]
 
+    def get_domain(self, kwargs):
+        domain = kwargs.get("domain", [])
+
+        # populate domain
+        if "from_date" in kwargs:
+            domain.append(("create_date", ">=", kwargs["from_date"]))
+            del kwargs["from_date"]
+
+        if "last_modified_date" in kwargs:
+            domain.append(("write_date", ">=", kwargs["last_modified_date"]))
+            del kwargs["last_modified_date"]
+
+        # Get all fields of model
+        model_fields = self.env[self.model].fields_get().keys()
+
+        kw_copy = kwargs.copy()
+        for field in kw_copy:
+            if field in model_fields:
+                domain.append((field, "=", kwargs[field]))
```

Comment:
@reichie020212 this won't work with x2many field

Reply:
@nhatnm0612 based on my understanding, we will only use many2one field here.

Reply:
thanks, @reichie020212 , resolved